cmd/juju/status/status.go

```diff
@@ -7,24 +7,27 @@ import (
 	"fmt"
 	"io"
 	"os"
-	"runtime"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/adrg/xdg"
 	"github.com/juju/clock"
 	"github.com/juju/cmd/v3"
 	"github.com/juju/collections/set"
 	"github.com/juju/errors"
 	"github.com/juju/gnuflag"
 	"github.com/juju/loggo"
+	"github.com/juju/viddy"
 
 	storageapi "github.com/juju/juju/api/client/storage"
 	jujucmd "github.com/juju/juju/cmd"
 	"github.com/juju/juju/cmd/juju/storage"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/juju/osenv"
 	"github.com/juju/juju/rpc/params"
+	"github.com/rivo/tview"
+	"github.com/spf13/viper"
```

Comment:
My concerns are that there is quite a bit more dependencies coming in than just viddy.
adrg/xdg, viddy, rivo/tviow, spf33/viper is quite a bit of new dependencies.
I'm much more in favor of "if `viddy` is in your path, use it, else use `watch` else complain that neither is installed.
You can always run `brew install watch` on Mac OSX, and it avoids having a large dependency and security patches, etc for the rest of juju to maintain these.

Reply:
And there is the fourth variant, taking into consideration that we use static config for Viddy, we can add an additional method to juju/viddy `PreConfiguedViddy` which will implement all stuff with config generation.  That will help to avoid adding deps in juju (when using juju/viddy)